### PR TITLE
Update to Pandell Shared Configuration

### PR DESCRIPTION
**Pull request by turnertime**
- Update gitignore to exclude install-state.gz
- Sync git attributes (updates line endings)
- Update to use [@pandell/prettier-config](https://github.com/pandell/frontend-config/tree/master/packages/prettier-config)
- Update to use [@pandell/typescript-config](https://github.com/pandell/frontend-config/tree/master/packages/typescript-config)
- Update to use [@pandell/eslint-config](https://github.com/pandell/frontend-config/pull/5) - flat config

### CODE REVIEW

**Review by milang — APPROVED** [review]
Biased review, I added some commits with additional changes. @turnertime please review.

File: .prettierignore [hunk 1]
```diff
@@ -1,5 +1,14 @@
-# output directories
-dist/
+# https://prettier.io/docs/en/ignore
+# spell-checker:words jison markdownlint
```
Comment (turnertime) [inline .prettierignore]: ⛏️ `jison` can be removed.

File: src/teamcity-reporter.ts [hunk 2]
```diff
@@ -1,83 +1,92 @@
...
-        const testFile = escapeForTeamCity(path.relative(test.context.config.rootDir, test.path));
-        console.log(`##teamcity[testSuiteStarted name='${testFile}']`);
+  /**
+   * Called with the result of every test file (also known as test suite).
+   *
+   * @see {@link https://www.jetbrains.com/help/teamcity/service-messages.html#Reporting+Tests} on reporting tests in TeamCity.
+   */
+  onTestResult(test: Test, testResult: TestResult): void {
+    const testFile = escapeForTeamCity(
+      path.relative(test.context.config.rootDir, test.path).replace("\\", "/"), // report paths using universal "/" separator, not platform-specific one
```
[the remark this answers is not in the document]
Comment (turnertime) [reply src/teamcity-reporter.ts]: Should update this to `replaceAll`?

**Review by turnertime — COMMENTED** [review]
@milang Nice improvements, thanks for the additions.

File: .prettierignore (hunk 1 above)
Comment (milang) [reply .prettierignore]: Removed.

File: src/teamcity-reporter.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (milang) [reply src/teamcity-reporter.ts]: Updated with global replace + extended tests to verify global replace.

**Comment by milang** [comment]
@turnertime addressed feedback, ready for next round.

**Review by turnertime — COMMENTED** [review]
Great stuff, thanks again.

**Comment by turnertime** [comment]
🧪 Verified the output matches the previous version in LandWorks Web with a couple of exceptions.

- duration (this is going to vary so I just normalized to `0`)
- separators (many were updated from `\` to `/` as expected)
- ordering varied slightly even when using `runInBand`

**Review by 101100 — APPROVED** [review]
Update looks good to me.  :cookie:

**Pull request merged**